Example/PaymentSheet Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -249,6 +252,7 @@ class PaymentSheetTestPlayground: UIViewController {
     var clientSecret: String?
     var ephemeralKey: String?
     var customerID: String?
+    var checkoutEndpoint: String?
```

Comment:
Make this a non optional and initialize it to the `defaultCheckoutEndpoint`

Reply:
Was hesitant about doing this at first but... Didn't. Thanks for calling this out!